cvat-sdk/gen/generate.sh

```diff
@@ -8,7 +8,7 @@ set -e
 
 GENERATOR_VERSION="v6.0.1"
 
-VERSION="2.0-alpha"
+VERSION="2.0a1"
```

Comment:
@zhiltsov-max , it is a known way to version python packages? It looks unusual.

Reply:
Yes, it is described here: https://peps.python.org/pep-0440/#public-version-identifiers